packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -181,6 +215,32 @@ function RecipientPicker(props: RecipientProps) {
     }
   }
 
+  const mapResolutionToRecipient = (resolution: NameResolution) => {
+    const lowerCaseAddress = resolution.address.toLowerCase()
+    switch (resolution.kind) {
+      case ResolutionKind.ADDRESS:
+        return getRecipientFromAddress(lowerCaseAddress, recipientInfo)
+      case ResolutionKind.NOM:
+        return {
+          address: lowerCaseAddress,
+          name: t('nomSpaceRecipient', { name: resolution.name ?? props.searchQuery }),
+        }
+      default:
+        return getRecipientFromAddress(lowerCaseAddress, recipientInfo)
+    }
+  }
+
+  const buildSections = (defaultSections: Section[]) => {
+    if (resolveKitCall.result && resolveKitCall.result.resolutions.length > 0) {
+      return [
+        ...defaultSections,
+        { key: t('others'), data: resolveKitCall.result.resolutions.map(mapResolutionToRecipient) },
```

Comment:
So, which are the `defaultSections`? If they include all the contacts then nom names will show up last and if a user has many contacts (which a lot of people do) they would never see it I think?

Hmm, maybe this isn't an issue because if a nom name shows up a user already typed, so most contacts will be filtered. Nevermind, my bad. Sending the message anyways just in case.

Reply:
Yep, `defaultSections` are the ones that are sent from the `Sent` component.
`Others` will appear after the contacts. But as you said, if you are already typing a `nom` or an `address` the contact list will be shorter.
Other easy option to implement is to have the list of `others` first. But I think `recent` and `contacts` are more important.